### PR TITLE
Minor tweaks after preparing VPA 0.14.0 release.

### PR DESCRIPTION
**Pull request by kgolab**
#### What type of PR is this?

/kind cleanup
/kind documentation

#### What this PR does / why we need it:

Some minor updates to the instructions on how to release VPA, based on the process done under #5851:
* use registry.k8s.io rather than k8s.gcr.io,
* make the copy&paste easier (no need to remove `$` manually)
* add a note about `add-version.sh` script

### CODE REVIEW

File: vertical-pod-autoscaler/RELEASE.md [hunk 1]
```diff
@@ -68,40 +68,44 @@ for component in recommender updater admission-controller ; do TAG=`grep 'const
...
     ```
 
 ## Promote image
 
 To promote image from staging repo send out PR updating
-[autoscaling images mapping](https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-autoscaling/images.yaml)
+[autoscaling images mapping](https://github.com/kubernetes/k8s.io/blob/master/registry.k8s.io/images/k8s-staging-autoscaling/images.yaml)
 ([example](https://github.com/kubernetes/k8s.io/pull/1318)).
 
+NOTE: You can use the [add-version.sh
```
Comment (jbartosik) [inline vertical-pod-autoscaler/RELEASE.md:98]: optional: I'd maybe use stronger wording, I expect the script to be less error prone than humans.
Maybe s/can/should/

**Review by jbartosik — APPROVED** [review]
Looks good to me, one small comment, not giving LGTM to avoid merge before you see the comment

File: vertical-pod-autoscaler/RELEASE.md (hunk 1 above)
Comment (kgolab) [reply vertical-pod-autoscaler/RELEASE.md]: Updated, PTAL.

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Pull request merged**